app/components/TransUnitFilter/index.jsx

```diff
@@ -0,0 +1,131 @@
+    resetFilter: React.PropTypes.func.isRequired,
+
+    // TODO replace with dispatched event
+    onFilterChange: React.PropTypes.func.isRequired,
+
+    // DO NOT RENAME, the translation string extractor looks specifically
+    // for gettextCatalog.getString when generating the translation template.
+    gettextCatalog: React.PropTypes.shape({
+      getString: React.PropTypes.func.isRequired
+    }).isRequired
+  },
+
+  getDefaultProps: () => {
+    return {
+      counts: {
+        total: 0,
+        approved: 0,
+        translated: 0,
+        needswork: 0,
+        untranslated: 0
+      }
+    };
+  },
+
+  render: function() {
+    return (
+      <ul className="u-listHorizontal u-sizeHeight-1">
+        <li className="u-sm-hidden u-sMV-1-4">
+          <FilterToggle
+            id="filter-phrases-total"
+            className="u-textSecondary"
+            isChecked={this.props.filterStatus.all}
+            onChange={this.props.resetFilter}
+            title={this.props.gettextCatalog.getString('Total Phrases')}
+            count={this.props.counts.total}
+            withDot={false}/>
+        </li>
+        <li className="u-ltemd-hidden u-sMV-1-4">
+          <FilterToggle
+            id="filter-phrases-approved"
+            className="u-textHighlight"
+            isChecked={this.props.filterStatus.approved}
+            onChange={this.props.onFilterChange.bind(undefined, 'approved')}
+            title={this.props.gettextCatalog.getString('Approved')}
+            count={this.props.counts.approved}/>
+        </li>
+        <li className="u-ltemd-hidden u-sMV-1-4">
+          <FilterToggle
+            id="filter-phrases-translated"
+            className="u-textSuccess"
+            isChecked={this.props.filterStatus.translated}
+            onChange={this.props.onFilterChange.bind(undefined, 'translated')}
+            title={this.props.gettextCatalog.getString('Translated')}
+            count={this.props.counts.translated}/>
+        </li>
+        <li className="u-ltemd-hidden u-sMV-1-4">
+          <FilterToggle
+            id="filter-phrases-needs-work"
+            className="u-textUnsure"
+            isChecked={this.props.filterStatus.needsWork}
+            onChange={this.props.onFilterChange.bind(undefined, 'needsWork')}
+            title={this.props.gettextCatalog.getString('Needs Work')}
+            count={this.props.counts.needswork}/>
+        </li>
+        <li className="u-ltemd-hidden u-sMV-1-4">
+          <FilterToggle
+            id="filter-phrases-untranslated"
+            className="u-textNeutral"
+            isChecked={this.props.filterStatus.untranslated}
+            onChange={this.props.onFilterChange.bind(undefined, 'untranslated')}
+            title={this.props.gettextCatalog.getString('Untranslated')}
+            count={this.props.counts.untranslated}/>
+        </li>
+  {/* A couple of parts of the Angular template that were not being used yet
```

Comment:
Yeah, I figured the easiest path here was just to keep the Angular templates since they have all the detail, and it saved spending any time migrating them just yet. Should be easy to convert them to React when needed.